docker-compose.yaml

```
@@ -77,6 +77,24 @@ services:
       - ropewiki_proxy_certs:/etc/letsencrypt
 #    restart: always
 
+  ropewiki_mailserver:
+    image: ropewiki/mailserver
+    build:
+      context: .
+      dockerfile: mailserver/Dockerfile
+    logging:
+      driver: json-file
+      options:
+        max-size: 10m
+    ports:
+      - "25:25"
+    environment:
+      - POSTFIX_myhostname=ropewiki.com  # who we identify ourselves as to clients
```

Can we pull this from the config like the other containers?

I'm not sure what a good default would be, but we probably don't want dev instances accidentally representing themselves as ropewiki.com

```suggestion
      - POSTFIX_myhostname=${WG_HOSTNAME:-localhost}  # who we identify ourselves as to clients
```

---

The domain in this case is just a "hello" message to clients (for their logs). It doesn't extend meaning any further to the rest of the process (e.g. won't influence which instance clients connect to).

---

Even if it's just a log message, it seems like we'd want the most correct value here, which is easily available with ${WG_HOSTNAME}.  Is there a reason to keep it as ropewiki.com?